app/models/user.rb

```diff
@@ -1,5 +1,6 @@
 class User < ApplicationRecord
 has_many :articles
+has_many :messages
```

Comment:
Połaczenie user-messages nie było nam potrzebne w aplikacji. Każdy użytkownik po podaniu maila w polu mógł wysłać wiadomość. Nie jest to złe, ale na przyszłość realizujemy zadania zgodnie z description.